Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.yml

```diff
@@ -77,6 +76,13 @@ args:
   name: phishingLabels
   required: false
   secret: false
+- default: false
+  defaultValue: '0.8'
+  description: The rario of the training set out of the entire data. Used for model evaluation
```

Comment:
The ratio of the training set to the entire data set, which is used for model evaluation.